### DISCUSSION TITLE
Install issue

### DISCUSSION THREAD

**Discussion by stevewa2066** [cariboulabs/cariboulite#74] [General]
Hi,
Started the install script on my PI 3 with 64 bit bullseye.

Got to the point where it is compiling SoapySDR. It stops..

![image](https://user-images.githubusercontent.com/65192818/221750766-95c6019f-91c5-42f4-8338-e948c6f8aa70.jpeg)

Any suggestions?

Steve

**Comment by kbthomas59** [comment]
I installed Soapy and Soapy remote before attempting to install Caribou.  What I got is identified and presented by SopySDRServer and can be used with gnuradio and CubicSDR.  That said, I only see noise, no signal anywhere with either SDR that the device appears as. No errors seen in the terminals as it runs.  It is as if it does not tune.

**Reply by ImDroided** [reply]
what is the output of sudo SoapySDRUtil --info

**Comment by pierre-olivier832** [comment]
I have the same problem , soapysdrutils is missing.

**Comment by ImDroided** [comment]
both of you should follow my guide here #94 

Start by deleting your caribou directory and starting fresh. I wrote this for DragonOS but I can confirm the same steps working with the stock OS minus the config.txt which will be located at /boot/config.txt

**Reply by jara8462** [reply]
how do you get the caribou/s ip address, have not found anything about it in all my searches. As you say in yourguide.

**Reply by ImDroided** [reply]
Well I log into my router. My router tells me all the devices and because I set the hostname to caribou it shows up in my router as the caribou. I can even use that name instead of an IP address. So for example I can go into my web browser just type in caribou and it will take me to my caribou.
